.github/workflows/check-pr-title.yaml

```
@@ -36,11 +36,41 @@ jobs:
       - name: Check PR title length
         if: github.event.pull_request.user.login != 'dependabot[bot]'
         env:
-          LENGTH: '70'
+          LENGTH: "70"
         run: |
           TITLE_LENGTH="$(echo -nE "$PR_TITLE" | wc --chars)"
           if [ "$TITLE_LENGTH" -gt "$LENGTH" ]
           then
             echo "::error::Please shorten the PR title from $TITLE_LENGTH to at most $LENGTH characters."
             exit 1
           fi
+
+  check-description:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+      - name: Check PR description
+        env:
+          PR_DESCRIPTION: ${{ github.event.pull_request.body }}
+        run: |
+          # shellcheck disable=SC2016
+          if [[ "$PR_DESCRIPTION" = *'- [x] `CHANGELOG.md` '* ]]; then
+            # shellcheck disable=SC2016
+            echo '`CHANGELOG.md` task is checked.'
+          else
+            # shellcheck disable=SC2016
+            echo '::error::`CHANGELOG.md` task is not checked.'
+            exit 1
+          fi
+          if [[ "$PR_DESCRIPTION" = *'- [x] documentation '* ]]; then
+            echo "documentation task is checked."
+          else
+            echo '::error::documentation task is not checked.'
+            exit 1
+          fi
+          if [[ "$PR_DESCRIPTION" = *'- [x] inspected CI results'* ]]; then
+            echo 'inspected CI results task is checked.'
+          else
+            echo '::error::inspected CI results task is not checked.'
+            exit 1
+          fi
```

```suggestion
          for CHECK_PREFIX in CHANGELOG Documentation "inspected CI results"; do
            if [[ "$PR_DESCRIPTION" = *"- [x] $CHECK_PREFIX"* ]]; then
              echo "$CHECK_PREFIX task is checked."
            else
              echo "::error::$CHECK_PREFIX task is not checked."
              exit 1
            fi
          done
```
Note: I'm not sure the syntax does work, but if it does, it would make things more compact

---

Improved.